src/types.tsx

```diff
@@ -913,3 +913,244 @@ export interface GestureProviderProps extends GestureProps {
   children?: React.ReactNode;
   gestureDetectorBridge: React.MutableRefObject<GestureDetectorBridge>;
 }
+
+// SplitView
+
+export type SplitViewDisplayModeButtonVisibility =
+  | 'always'
+  | 'automatic'
+  | 'never';
+
+export type SplitViewSplitBehavior =
+  | 'automatic'
+  | 'displace'
+  | 'overlay'
+  | 'tile';
+
+export type SplitViewPrimaryEdge = 'leading' | 'trailing';
+
+export type SplitViewDisplayMode =
+  | 'automatic'
+  | 'secondaryOnly'
+  | 'oneBesideSecondary'
+  | 'oneOverSecondary'
+  | 'twoBesideSecondary'
+  | 'twoOverSecondary'
+  | 'twoDisplaceSecondary';
+
+export interface SplitViewColumnMetrics {
+  /**
+   * Specifies the minimum width for the primary column in the SplitView layout, typically representing the leftmost sidebar.
+   *
+   * @platform ios
+   */
+  minimumPrimaryColumnWidth?: number;
+  /**
+   * Specifies the maximum width for the primary column in the SplitView layout, typically representing the leftmost sidebar.
+   *
+   * @platform ios
+   */
+  maximumPrimaryColumnWidth?: number;
+  /**
+   * Specifies the preferred width for the primary column in the SplitView layout, typically representing the leftmost sidebar.
+   *
+   * @platform ios
+   */
+  preferredPrimaryColumnWidth?: number;
+  /**
+   * Specifies the minimum width for the supplementary column in the SplitView layout, typically representing the intermediate sidebar.
+   *
+   * @platform ios
+   */
+  minimumSupplementaryColumnWidth?: number;
+  /**
+   * Specifies the maximum width for the supplementary column in the SplitView layout, typically representing the intermediate sidebar.
+   *
+   * @platform ios
+   */
+  maximumSupplementaryColumnWidth?: number;
+  /**
+   * Specifies the preferred width for the supplementary column in the SplitView layout, typically representing the intermediate sidebar.
+   *
+   * @platform ios
+   */
+  preferredSupplementaryColumnWidth?: number;
+
+  // iOS 26 only
+  /**
+   * Specifies the minimum width for the secondary column in the SplitView layout, typically for the view with the main content.
+   *
+   * @platform iOS 26+
+   * @supported iOS 26 or higher
```

Comment:
```suggestion
   * @platform ios
   * @supported iOS 26 or higher
```
If we're using `@supported` to note supported versions, maybe we should leave `@platform` without version information?

Reply:
https://github.com/software-mansion/react-native-screens/pull/3034/commits/9f56c648979add47c959586c391e8864a719a1d9